python/lsst/meas/base/catalogCalculation.py

```diff
@@ -103,9 +103,9 @@ def __exit__(self, exc_type, exc_value, traceback):
 
 class CatalogCalculationConfig(lsst.pex.config.Config):
     '''
-    Default CatalogCalculationConfig. Currently this is an empty list, meaning that there are no default
```

Comment:
This doc string did not get updated when a default plugin was added (on 5877 I think) Im sorry I didn't catch it, but am happy you did